internal/server/iigointernal/monitoring.go

```diff
@@ -42,6 +44,10 @@ func (m *monitor) monitorRole(roleAccountable baseclient.Client) shared.MonitorR
 			evaluationResult = m.evaluateCache(roleToMonitor, m.gameState.RulesInfo.CurrentRulesInPlay)
 		}
 
+		if !evaluationResult {
+			m.gameState.IIGOTurnsInPower[roleName] = m.config.IIGOConfig.IIGOTermLengths[roleName]
```

Comment:
```suggestion
			m.gameState.IIGOTurnsInPower[roleName] = m.config.IIGOConfig.IIGOTermLengths[roleName] + 1
```
for term to have 'ended' when cheating is found

Reply:
Ok cheers, I'll commit once I've cleared a different bug.